5.0/en/0x14-V6-Cryptography.md

```diff
@@ -62,10 +62,9 @@ Cryptographically secure Pseudo-random Number Generation (CSPRNG) is incredibly
 
 | # | Description | L1 | L2 | L3 | CWE |
 | :---: | :--- | :---: | :---: | :---: | :---: |
-| **6.3.1** | [MODIFIED] All random numbers and strings intended to be non-guessable must be generated using a CSPRNG. | ✓ | ✓ | ✓ | 338 |
-| **6.3.2** | 6.3.2 [MODIFIED] Verify that GUIDs are created with an implementation of the GUID v4 or v7 algorithms which utilizes a CSPRNG.                                               |     |  ✓  |  ✓  | 338 |
-| **6.3.3** | Random number generation must work properly under heavy system load, or the system must degrade gracefully. | ✓ | ✓ | ✓ | 338 |
-
+| **6.3.1** | [GRAMMAR, LEVEL L2 > L1] Verify that all random numbers and strings which are intended to be non-guessable must be generated using a cryptographically-secure pseudo-random number generator (CSPRNG). | ✓ | ✓ | ✓ | 338 |
+| **6.3.2** | [MODIFIED] Verify that GUIDs are created with an implementation of the GUID v4 or v7 algorithms which utilizes a cryptographically-secure pseudo-random number generator (CSPRNG). | | ✓ | ✓ | 338 |
```

Comment:
Shouldn't we use UUID here?

Reply:
GUID is often used synonymously with UUID but is historically tied to use within a Microsoft world. If we were to be pedantic, while GUID is functionally equivalent in many respects, UUID is the term formally defined by the broader industry standard (RFC 4122), which is why maybe using UUID is generally recommended for clarity and consistency. 

I'm easy either way here

Reply:
UUID seems the more generic so I think I prefer that